hieradata_aws/integration.yaml

```diff
@@ -136,6 +136,7 @@ govuk_jenkins::jobs::search_test_spelling_suggestions::cron_schedule: '0 10 * *
 govuk_jenkins::jobs::search_api_fetch_analytics_data::skip_page_traffic_load: true
 govuk_jenkins::jobs::search_api_fetch_analytics_data::cron_schedule: '30 9 * * 1-5'
 
+govuk_jenkins::jobs::search_relevancy_rank_evaluation::cron_schedule: 'H */3 * * *' # every three hours
```

Comment:
Do we know what happens if the rank evaluation runs during the data sync?  Will it just work, or will it throw an error if it overlaps with the snapshot restore?

Reply:
As you suggested, it'll run every 45 mins to avoid coinciding with the indices being deleted:

https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/class/staging/search.yaml#L19